src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -125,7 +125,7 @@ csm::ImageCoord UsgsAstroFrameSensorModel::groundToImage(
 
   // Camera rotation matrix
   double m[3][3];
-  calcRotationMatrix(m,adjustments);
+  calcRotationMatrix(m, adjustments);
```

Comment:
Thank you for always fixing these silly little things you find!

Reply:
:+1: :)